cla_public/templates/contact.html

```diff
@@ -154,26 +154,23 @@ <h1 class="govuk-heading-xl">{{ title }}</h1>
   </script>
 
   <script type="text/html" id="postcodeNotEnteredText">
-    <div class="form-row field-error">
-      <p>
-        {{ _('Must contain a valid postcode') }}
-      </p>
-    </div>
+    <span class="govuk-error-message">
+      <span class="govuk-visually-hidden">{% trans %}Error{% endtrans %}:</span>
+      <span class="cla-error-message">{{ _('Must contain a valid postcode') }}</span>
+    </span>
   </script>
 
   <script type="text/html" id="noAddressesFoundText">
-    <div class="form-row field-error">
-      <p>
-        {{ _('No addresses were found with that postcode, but you can still enter your address manually') }}
-      </p>
-    </div>
+    <span class="govuk-error-message">
+      <span class="govuk-visually-hidden">{% trans %}Error{% endtrans %}:</span>
+      <span class="cla-error-message">{{ _('No addresses were found with that postcode, but you can still enter your address manually') }}</span>
+    </span>
   </script>
 
   <script type="text/html" id="requestFailedText">
-    <div class="form-row field-error">
-      <p>
-        {{ _('Request failed: ') }} <%= textStatus %>, <%= error %>
-      </p>
-    </div>
+    <span class="govuk-error-message">
+      <span class="govuk-visually-hidden">{% trans %}Error{% endtrans %}:</span>
+      <span class="cla-error-message">{{ _('Request failed: ') }} <%= textStatus %>, <%= error %></span>
```

Comment:
Why the switch between {% trans %} and {{ _('') }}

Reply:
No reason.  The {{ _('') }} approach was already there - I just added stuff around it.  
The {% trans %} approach was copied from elsewhere.

Reply:
Changed to `{{ _('') }}`